spacy/pipeline/textcat.py

```diff
@@ -294,7 +300,7 @@ def _examples_to_truth(
             for j, label in enumerate(self.labels):
                 if label in eg.reference.cats:
                     truths[i, j] = eg.reference.cats[label]
-                else:
+                elif self.support_missing_values:
```

Comment:
What is going on here is kind of subtle and might merit a comment. Something like:

> this is always false by default, but subclasses override it. For the single class case, other values will just be ignored.

Reply:
I've added the comment to the definition of the property itself, do you think that's sufficient?

Reply:
That should do it, thanks!